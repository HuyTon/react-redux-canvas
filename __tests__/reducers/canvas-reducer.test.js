// Reducer to be tested
import canvasReducer from '../../src/client/reducers/canvas-reducer';
import {
    GET_CANVAS_SUCCESS,
} from '../../src/client/actions/action-types';

describe('GET_CANVAS_SUCCESS', () => {
    test('is correct', () => {
        const action = { type: GET_CANVAS_SUCCESS };

        expect(canvasReducer(undefined, action)).toMatchSnapshot();
    });
});