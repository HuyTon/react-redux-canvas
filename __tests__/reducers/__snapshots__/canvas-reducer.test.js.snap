// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET_CANVAS_SUCCESS is correct 1`] = `
Object {
  "canvas": undefined,
}
`;
